src/main/java/com/puppycrawl/tools/checkstyle/checks/design/InnerTypeLastCheck.java

```diff
@@ -127,7 +127,7 @@ public void visitToken(DetailAST ast) {
             rootClass = false;
         }
         else {
-            DetailAST nextSibling = ast.getNextSibling();
```

Comment:
This is connected to `CLASS_MEMBER_TOKENS`.

`ScopeUtil.isInCodeBlock` passes but `CLASS_MEMBER_TOKENS` has the specific tokens: VARIABLE_DEF, METHOD_DEF, CTOR_DEF, INSTANCE_INIT, STATIC_INIT, COMPACT_CTOR_DEF. However `ast` is from the visit method, and it only stops on CLASS_DEF, INTERFACE_DEF, RECORD_DEF before going on to the siblings.

Since neither list ever has a matching token, it means this mutation can never be killed.